src/components/timecard/shift/Shift.spec.jsx

```diff
@@ -0,0 +1,199 @@
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+import { act } from 'react-test-renderer';
+import { renderWithApplicationContext } from '../../../test/helpers/TestApplicationContext';
+import Shift from '../shift/Shift';
+
+jest.mock('../../../api/services/timecardService');
+
+let existingTimeEntry;
+beforeEach(() => {
+  existingTimeEntry = {
+    timeEntryId: '00000000-0000-0000-0000-000000000001',
+    startTime: '2022-11-01T08:00:00Z',
+    finishTime: '2022-11-01T16:00:00Z',
+    timePeriodTypeId: '00000000-0000-0000-0000-000000000001',
+  };
+});
+
+const newTimeEntry = {
+  timePeriodTypeId: '00000000-0000-0000-0000-000000000001',
+};
+
+describe('Shift', () => {
+  it('should display a summary list with titles for time period, hours and meal break', () => {
+    renderWithApplicationContext(
+      <Shift
+        summaryErrors={[]}
+        timeEntry={existingTimeEntry}
+        timeEntriesIndex={0}
+      />
+    );
+
+    const shiftDetailsTitles = ['Time period', 'Hours', 'Meal break'];
+
+    shiftDetailsTitles.forEach((title) => {
+      expect(screen.getByText(title)).toBeTruthy();
+    });
+  });
+
+  it('should show EditShiftHours component when first directed to page', async () => {
```

Comment:
There are a few references to EditShiftHours - could we change them to EditShift?